.github/ISSUE_TEMPLATE/release-runtime.md

```diff
@@ -27,20 +27,21 @@ These checks should be performed on the codebase.
 The following checks can be performed after we have forked off to the release-
 candidate branch or started an additional release candidate branch (rc-2, rc-3, etc)
 
-- [ ] Verify [new migrations](../../docs/release.md#new-migrations) complete successfully, and the
+- [ ] Verify [new migrations](https://github.com/paritytech/cumulus/blob/master/docs/release.md#new-migrations) complete successfully, and the
     runtime state is correctly updated for any public (non-private/test)
     networks
-- [ ] Run integration tests
+- [ ] Verify new extrinsics have been correctly whitelisted/blacklisted for [proxy filters](https://github.com/paritytech/cumulus/blob/master/docs/release.md#proxy-filtering).
```

Comment:
is this even still relevant?

Reply:
yep

Reply:
I would say no.

There were discussionsabout that and the conclusion is that no error can result in a risk beyond lacking a feature and in that case, it can be fixed in a new runtime upgrade.

So the dev should be careful when adding/removing calls and ensure the proxies are fixed accordingly but it does not make much sense to double check that during the release process.

The output of the "Extrinsic Ordering" can hint however if a new call was inrtoduced.

The Proxy check has been removed and should not be added again, at least not in the release checklist.